lib/access_control.js

```diff
@@ -0,0 +1,64 @@
+const {
+  useAccessControl
+} = require('../config/email_constants');
+
+
+const requestAuthReduce = (request) => {
```

Comment:
Instead of `requestAuthReduce()` consider `getAuthzFlags()` or something. `Reduce` threw me off, and it is not requesting something but instead looking at the request object.